template.go

```diff
@@ -45,9 +45,13 @@ https://github.com/{{.GithubRepo}}/issues.
 {{range $project := .Changes}}
 
 ### Changes{{if $project.Name}} from {{$project.Name}}{{end}}
+<details><summary>{{len $project.Changes}} commits</summary>
```

Comment:
if `len` is 1, no `s` on commit

Reply:
Won't be 1 because we have merge commits

Reply:
I agree, but still better not to make that assumption for all projects which use this tool.

Reply:
updated